app/export.go

```diff
@@ -123,19 +120,18 @@ func (app *Quicksilver) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAdd
 	for _, del := range dels {
 		valAddr, err := sdk.ValAddressFromBech32(del.ValidatorAddress)
 		if err != nil {
-			return err
-		}
-		delAddr, err := sdk.AccAddressFromBech32(del.DelegatorAddress)
-		if err != nil {
-			return err
+			panic(err)
```

Comment:
Why are we using `log.Fatal` and `panic` for some of these errors?  If we are going to terminate the program on error from this function, we should just pick one consistent way of doing it.

Reply:
Same here: https://github.com/ingenuity-build/quicksilver/pull/420/files/950d879a0ef010866226d94c09b3f73e7ce5adde#r1197738404